### PR TITLE
Improvement: Feedback #3002

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [x] Improvement (make documentation even better)
- [x] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

This PR improves the documentation for migrating to the new version of GTM. Previously, the documentation was focused on UA, which is now outdated, considering the new app version that supports GA4. Therefore, this update removes the obsolete content and provides relevant information for the GA4 migration process.

### CODE REVIEW

**Review by filipewl — APPROVED** [review]
Looks good to me! But it would be nice to have the eyes of Raquel Tótora on this.

**Comment by queltotora1** [comment]
We don't need to teach the clients how to create variables for the UA implementation, but I believe it's still important to keep the information about the differences between the ecommerce and ecommerceV2 objects. Both of them will still be available regardless of the GA4/UA implementation and the data can be used by our clients. That's the only thing I would make it different.

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 1]
```diff
@@ -1,96 +1,45 @@
...
 seeAlso:
  - "/docs/guides/vtex-google-tag-manager"
  - "/docs/guides/vtex-io-documentation-setting-up-google-tag-manager"
 ---
 
-Version 3.x of VTEX’s Google Tag Manager (GTM) app tracks the entire user’s journey through the store, from viewing a product to purchasing it, via events triggered by the variable `ecommerceV2`.
+Version 3.x of VTEX’s Google Tag Manager (GTM) app enables comprehensive tracking of the user's journey within the store, from product views to purchases, using events triggered by the `ecommerceV2` variable. Additionally, it allows stores to send events in both the Google Analytics 4 (GA4) format and the Universal Analytics (UA) format, facilitating a seamless transition to GA4.
 
-New and rearranged properties were added to the product schema to represent on the events and to track the user's journey and keep product information consistent across all areas of the store.
+## New properties and product schema
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:15]:
```suggestion
## New properties to the product schema
```

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 2]
```diff
@@ -1,96 +1,45 @@
...
+updatedAt: "2023-05-17T09:17:44.057Z"
 category: "Storefront Development"
-excerpt: "Step-by-step to update the Google Tag Manager app to version 3.x."
 seeAlso:
  - "/docs/guides/vtex-google-tag-manager"
  - "/docs/guides/vtex-io-documentation-setting-up-google-tag-manager"
 ---
 
-Version 3.x of VTEX’s Google Tag Manager (GTM) app tracks the entire user’s journey through the store, from viewing a product to purchasing it, via events triggered by the variable `ecommerceV2`.
+Version 3.x of VTEX’s Google Tag Manager (GTM) app enables comprehensive tracking of the user's journey within the store, from product views to purchases, using events triggered by the `ecommerceV2` variable. Additionally, it allows stores to send events in both the Google Analytics 4 (GA4) format and the Universal Analytics (UA) format, facilitating a seamless transition to GA4.
```
Comment (carolinamenezes) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:13]:
```suggestion
Version 3.x of [VTEX Google Tag Manager (GTM) app](https://developers.vtex.com/docs/apps/vtex.google-tag-manager) introduces enhanced tracking capabilities for user interactions within the store. With the integration of the `ecommerceV2` variable, this version allows for comprehensive monitoring of the user journey, capturing events from product views to completed purchases. Additionally, it allows stores to send events in both the Google Analytics 4 (GA4) format and the Universal Analytics (UA) format, facilitating a seamless transition to GA4.
```

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 3]
```diff
@@ -1,96 +1,45 @@
...
 
-New and rearranged properties were added to the product schema to represent on the events and to track the user's journey and keep product information consistent across all areas of the store.
+## New properties to the product schema
```
Comment (carolinamenezes) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:15]:
```suggestion
## Product schema properties of GTM 3.x 
```

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 4]
```diff
@@ -1,96 +1,45 @@
...
-| `brand`          | Unchanged property |
-| `dimension1`     | New property       |
-| `dimension2`     | New property       |
-| `dimension3`     | New property       |
+To ensure consistency in product information across all areas of the store and effectively track the user's journey, Google Tag Manager 3.x introduces Enhanced Ecommerce properties to the product data schema as events. By adding these properties, stores can include additional information such as product printing, promotion, and sales data, thereby enhancing the overall tracking capabilities and maintaining a unified view of the user's journey throughout the store.
 
-> ℹ️ For more details on these properties, go to the section [Product schema](#product-schema)
+| GTM 3.x property | Type               | Description                                                                 |
+| ---------------- | ------------------ | --------------------------------------------------------------------------- |
+| `id`             | Renewed property   | Product ID - Previously SKU ID.                                             |
```
Comment (carolinamenezes) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:21]: O que quer dizer com `renewed`?

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 5]
```diff
@@ -1,96 +1,45 @@
...
-| dimension2 | SKU Reference ID.                                                           |
-| dimension3 | SKU Name (does not include the Product Name).                               |
+## Migrating to GTM 3.x
 
-The `dimension1`, `dimension2`, `dimension3` properties are custom dimensions that you can use to collect and analyze data that Google Analytics does not automatically create.
+ To migrate to the app's 3.x version, please follow these guides:
 
-For more information about custom dimensions and Enhanced Ecommerce, refer to [Custom dimensions and metrics](https://support.google.com/analytics/answer/2709828?hl=en&ref_topic=2709827#configuration&zippy=%2Cin-this-article) and [Google Enhanced ecommerce official guide](https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#ecommerce-data) respectively.
+1. [Create a Google Analytics 4 property](https://developers.vtex.com/docs/guides/vtex-io-documentation-installing-google-tag-manager#create-a-google-analytics-4-property): Refer to the section Create a Google Analytics and follow the instructions provided.
```
Comment (carolinamenezes) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:42]:
```suggestion
1. [Create a Google Analytics 4 property](https://developers.vtex.com/docs/guides/vtex-io-documentation-installing-google-tag-manager#create-a-google-analytics-4-property): Refer to the "Create a Google Analytics 4 property" section and follow the instructions provided.
```

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 6]
```diff
@@ -1,96 +1,45 @@
...
+2. Once you have completed the setup, proceed to the  [Setting up Google Tag Manager documentation](https://developers.vtex.com/docs/guides/vtex-io-documentation-setting-up-google-tag-manager) documentation for instructions on configuring events, tags, and triggers specifically for GA4.
```
Comment (carolinamenezes) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:43]:
```suggestion
2. Once you have completed the setup, proceed to the [Setting up Google Tag Manager](https://developers.vtex.com/docs/guides/vtex-io-documentation-setting-up-google-tag-manager) guide for instructions on configuring events, tags, and triggers specifically for GA4.
```

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md (hunk 4 above)
Comment (Mariana-Caetano) [reply docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:21]: São props que já existiam, mas foram atualizadas para essa versão. Pensando bem, vou mudar para "updated"

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 7]
```diff
@@ -1,96 +1,45 @@
...
-| `brand`          | Unchanged property |
-| `dimension1`     | New property       |
-| `dimension2`     | New property       |
-| `dimension3`     | New property       |
+To ensure consistency in product information across all areas of the store and effectively track the user's journey, Google Tag Manager 3.x introduces Enhanced Ecommerce properties to the product data schema as events. By adding these properties, stores can include additional information such as product printing, promotion, and sales data, thereby enhancing the overall tracking capabilities and maintaining a unified view of the user's journey throughout the store.
 
-> ℹ️ For more details on these properties, go to the section [Product schema](#product-schema)
+| GTM 3.x property | Type               | Description                                                                 |
+| ---------------- | ------------------ | --------------------------------------------------------------------------- |
+| `id`             | Renewed property   | Product ID - Previously SKU ID.                                             |
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:21]:
```suggestion
| `id`             | Updated property   | Product ID - Previously SKU ID.                                             |
```

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 8]
```diff
@@ -1,96 +1,45 @@
...
+| `variant`        | Renewed property   | SKU ID - Previously SKU Name. The variant of the product, e.g., Rebel pink. |
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:22]:
```suggestion
| `variant`        | Updated property   | SKU ID - Previously SKU Name. The variant of the product, e.g., Rebel pink. |
```

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md [hunk 9]
```diff
@@ -1,96 +1,45 @@
...
+| `name`           | Renewed property   | Product Name - Previously Product Name or SKU Name.                         |
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md:23]:
```suggestion
| `name`           | Updated property   | Product Name - Previously Product Name or SKU Name.                         |
```

**Review by carolinamenezes — APPROVED** [review]

File: docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md (hunk 4 above)
Comment (carolinamenezes) [reply docs/vtex-io/Storefront-Guides/google-tag-manager/vtex-io-documentation-migrating-google-tag-manager-app.md]: Boa! Pensei em updated mesmo

**Pull request merged**